db/schema.rb

```diff
@@ -21,6 +21,7 @@
     t.float    "latitude"
     t.float    "longitude"
     t.string   "address",    limit: 255
+    t.boolean  "reviewed"
```

Comment:
Can you make the default be set to false? This will make it easier, so when people create entries, they are all unreviewed.